data/talkactions/scripts/looktype.lua

```diff
@@ -14,7 +29,7 @@ function onSay(player, words, param)
 	end
 
 	local lookType = tonumber(param)
-	if lookType >= 0 and lookType < 700 and not table.contains(invalidTypes, lookType) then
+	if lookType >= 0 and lookType < 903 and not table.contains(invalidTypes, lookType) then
```

Comment:
If all lookTypes > 903 are invalid, then why are you adding a lot of >700 IDs to the list above?

Reply:
He is back from the future. x)